tests/integration/vlan_test.py

```diff
@@ -45,6 +45,40 @@ def test_add_and_remove_two_vlans_on_same_iface(eth1_up):
     assert not current_state[INTERFACES]
 
 
+def test_rollback_for_vlans(eth1_up):
+    current_state = statelib.show_only([])
```

Comment:
Is there no method to show the full state? Passing an empty list here looks strange. This is also true for the other instance of this.